dashboard/app/models/lesson.rb

```diff
@@ -786,10 +786,10 @@ def lesson_plan_has_verified_resources
   # - be migrated
   # - be in a course version
   # - be in course versions from the same version year
-  def copy_to_unit(destination_unit, new_level_suffix = nil)
+  def copy_to_unit(destination_unit, new_level_suffix = nil, destination_professional_learning_course = nil)
```

Comment:
it's a bit hard to tell what this parameter is doing. can you please update the contract to account for this parameter, including its type and what it does?